storage/src/backend/registry.rs

```diff
@@ -282,10 +304,62 @@ struct RegistryReader {
     blob_id: String,
     connection: Arc<Connection>,
     state: Arc<RegistryState>,
+    mirrors: Arc<Mirror>,
```

Comment:
The field name is `mirrors`, but the type `Arc<Mirror>` does not indicate its multitude